lab-andrew/route/brew-router.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+// node moudles
+// npm modules
+// app modules
+// globals && then modules dependent on globals
+const Router = require('express').Router;
+const debug = require('debug')('brew:brew-router');
+const jsonParser = require('body-parser').json();
+
+const AppError = require('../lib/app-error');
+const storage = require('../lib/storage');
+const Brewer = require('../model/brewer');
+// const newStorage = new Storage(`${__dirname}/../data`);
+
+// const response = new AppError();
+const brewRouter = module.exports = new Router();
+
+function createBrewer(reqBody) {
+  debug('createBrewer');
+  return new Promise(function(resolve, reject) {
+    var brewer;
```

Comment:
You should set this to null explicitly so if there is an issue with your assignment to the variable later you can have a better sense of what went wrong.